contracts/account/modules/ERC7579P256Validator.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {P256} from "@openzeppelin/contracts/utils/cryptography/P256.sol";
+import {ERC7579Validator} from "./ERC7579Validator.sol";
+
+/**
+ * @dev {ERC7579Validator} for {P256} signatures.
+ */
+abstract contract ERC7579P256Validator is ERC7579Validator {
+    mapping(address sender => bytes32) private _associatedQx;
+    mapping(address sender => bytes32) private _associatedQy;
```

Comment:
same logic as ERC7579ECDSA, I think we should work on the validator in a different PR.

But still here: 
```suggestion
    mapping(address sender => bytes32[2]) private _associatedQxQy;
```